date/date_c.php

```diff
@@ -313,15 +313,15 @@ public function __wakeup() { }
 class DateTime implements DateTimeInterface {
     /**
      * (PHP 5 &gt;=5.2.0)<br/>
-     * @link https://www.php.net/manual/en/datetime.construct.php
+     * @link https://secure.php.net/manual/en/datetime.construct.php
      * @param string $time [optional]
      * <p>A date/time string. Valid formats are explained in {@link www.php.net/manual/en/datetime.formats.php Date and Time Formats}.</p>
      * <p>
      * Enter <b>now</b> here to obtain the current time when using
      * the <em>$timezone</em> parameter.
      * </p>
      * @param DateTimeZone $timezone [optional] <p>
-     * A {@link https://www.php.net/manual/en/class.datetimezone.php DateTimeZone} object representing the
+     * A {@link https://secure.php.net/manual/en/class.datetimezone.php DateTimeZone} object representing the
```

Comment:
https://www.php.net/manual/en/class.datetimezone.php is also works but it's not navigatable as well as suggested url cause of https://youtrack.jetbrains.com/issue/WI-37030

Reply:
Updated url to not use secure subdomain, but still removed the www subdomain. httpS with www subdomain does NOT work.